src/Razor/test/testapps/SimpleMvc21/Views/Home/About.cshtml

```diff
@@ -1,4 +1,5 @@
-﻿@{
+﻿
```

Comment:
Was this an intentional change? We usually try to avoid making changes to these files as our functional tests unfortunately rely on some things being at particular offsets.

Reply:
Unintentional, removed, thanks.